src/components/space/functionality/ViewSelect.tsx

```diff
@@ -80,7 +80,7 @@ const ViewSelect = (): JSX.Element => {
         </FormControl>
       </Grid>
       <Grid item ml={2} mr={1}>
-        <Tooltip title={viewMessage}>
+        <Tooltip title={t(viewMessage)}>
```

Comment:
I would actually add the `t` higher, so it's clear what's the string you translate.